docs/modules/ROOT/pages/guides/testing_with_zombienet.adoc

```diff
@@ -95,7 +95,7 @@ cargo build --release --features="async-backing"
 +
 image::alice-direct-link.png[Alice Direct Link]
 
-. open the link with Chrome. [`polkadot.js.org/apps`](http://polkadot.js.org/apps) As of 2024 July, it doesn’t work on Safari and Brave. And it might be buggy in Firefox. So, open it with Chrome.
+. open the link with Chrome: link:https://polkadot.js.org/apps[PolkadotJS]. As of 2024 July, it doesn’t work on Safari and Brave. And it might be buggy in Firefox.
```

Comment:
```suggestion
Open the link with Chrome: link:https://polkadot.js.org/apps[PolkadotJS]. As of July 2024, it doesn’t work on Safari and Brave, and it might be buggy on Firefox.
```

Reply:
Left a few suggestions to improve writing. One questions: When we say Firefox might be buggy is it because we tried it and it was actually buggy? Or we haven't tried it so we are not sure? If it's the latter I would actually say that vs saying is buggy.

Reply:
we tested and it was buggy. Much less buggy than other browsers, but still buggy. So on the first trial, we thought firefox was working, but when trying to submit extrinsics and etc. it turned out to be buggy as well